tests/e2e/vc_reboot_volume_lifecycle.go

```diff
@@ -90,7 +90,8 @@ var _ bool = ginkgo.Describe("Verify volume life_cycle operations works fine aft
 			scParameters[scParamStoragePolicyID] = profileID
 			// create resource quota
 			createResourceQuota(client, namespace, rqLimit, storagePolicyName)
-		} else {
+		}
+		if guestCluster {
 			ginkgo.By("CNS_TEST: Running for GC setup")
 			scParameters[svStorageClassName] = storagePolicyName
 		}
```

Comment:
you can move this in the block where you are creating the SC

Reply:
done